frontend/src/components/ManageActivity.vue

```diff
@@ -110,6 +115,21 @@ export default {
       duration: 0
     };
   },
+
+  async created() {
+    await this.$nextTick();
```

Comment:
Why do we `await this.$nextTick()` twice here?

Reply:
Seems to be an issue with Quasar-vue framework, props are not immediately available in the created hook. Need to wait for 2 cycles for data to be updated. Will look for any other solutions if possible

Reply:
OK, no worries. Please just add a code comment with a link to the up-stream issue report.

Reply:
No need. I added a v-if in the calling component